CMakeLists.txt

```diff
@@ -505,14 +500,7 @@ if(TARGET zlib::zlib)
 	add_library(ZLIB::ZLIB ALIAS zlib::zlib)
 endif()
 
-option(FORCE_BUNDLED_MINIZIP "Force bundled Minizip library" OFF)
```

Comment:
Just in case - can somebody verify that mod installation works as expected on windows/msvc?